File: Coding/03_Beginner/Full_search/JavaScript/14.js
const fs = require("fs");
const S = fs.readFileSync(0, "utf8").trim();

let cnt = 0;
for (let i = 0; i < S.length - 1; i++){
    if (S[i] == S[i+1]){
        cnt++;
    }
}

console.log(cnt);

File: Coding/03_Beginner/Full_search/C++/14.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
    string S;
    cin >> S;
    int cnt = 0;

    for (int i = 0; i < (int)S.size()-1; i++){
        if (S[i] == S[i+1]){
            cnt++;
        }
    }

    cout << cnt << endl;
    return 0;
}